scripts/scil_evaluate_connectivity_pairwise_agreement_measures.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of connectivity matrix.
+
+The computed similarity measures are:
+sum of square difference and pearson correlation coefficent
+"""
+
+import argparse
+import itertools
+import json
+import logging
+import os
+import shutil
+
+import numpy as np
+
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             load_matrix_in_any_format)
+from scilpy.tractanalysis.reproducibility_measures import compute_dice_voxel
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_matrices', nargs='+',
+                   help='Path of the input bundles.')
+    p.add_argument('out_json',
+                   help='Path of the output json file.')
+    p.add_argument('--single_compare',
+                   help='Compare inputs to this single file.')
+
+    add_json_args(p)
+    add_overwrite_arg(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, args.in_matrices)
+    assert_outputs_exist(parser, args, [args.out_json])
+
+    all_matrices = []
+    for filename in args.in_matrices:
+        tmp_mat = load_matrix_in_any_format(filename)
+        all_matrices.append(tmp_mat / np.max(tmp_mat))
+
+    output_measures_dict = {'SSD': [], 'Correlation': []}
+    pairs = list(itertools.combinations(all_matrices, r=2))
+    for i in pairs:
+        ssd = np.sum((pairs[0] - pairs[1]) ** 2)
+        output_measures_dict['SSD'].append(ssd)
+        # only calculates for the first two arrays
+        corrcoef = np.corrcoef(all_matrices[0], all_matrices[1])
```

Comment:
This should be pairs, not all_matrices.
Also np.corrcoef expects a 1D array (everything we have is 2D) and returns a covariance matrix not a scalar.
(I think this code does not execute)